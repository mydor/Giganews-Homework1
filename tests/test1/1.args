user1@domain1.com
testpw1
