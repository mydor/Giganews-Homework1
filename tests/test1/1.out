Authentication successful for user1@domain1.com
User: user1
Domain: domain1.com
Username: user1@domain1.com
